tests/testPhotoCal.py

```diff
@@ -71,11 +81,34 @@ def setUp(self):
 
         self.refObjLoader = makeRefLoader()
 
+        self.res = self.getAstrometrySolution(loglvl=Log.DEBUG)
+        self.matches = self.res.matches
+
+        logLevel = Log.DEBUG
+        self.log = Log(Log.getDefaultLog(),
+                  'testPhotoCal',
+                  logLevel)
+
+        self.schema = self.matches[0].second.schema
+
+        self.config = PhotoCalConfig()
+
+        # The test and associated data have been prepared on the basis that we
+        # use the PsfFlux to perform photometry.
+        self.config.fluxField = "base_PsfFlux_flux"
+
+        self.config.doWriteOutput = False    # schema is fixed because we already loaded the data
+
```

Comment:
A bit too many blank lines. This can improve readability, but in this case I would say it reduces it due to fewer lines with content fitting on a screen.